toolchain/mfc/bench.py

```diff
@@ -116,6 +117,12 @@ def _lock_to_str(lock):
             if target.name not in lhs_summary or target.name not in rhs_summary:
                 continue
 
+            if float(f"{lhs_summary[target.name]}") <= 0:
+                raise MFCException(f"Non-positive runtime in LHS - Case: {slug} - Target: {target.name}")
```

Comment:
With this implementation, if even a single table element cannot be calculated, nothing will be printed other than this error. It would be nice to still print the table: output warnings while calling `add_row` so (change `raise MFCException` to `cons.print`) and then raise an exception after printing the table.